draft-ietf-scone-protocol.md

```diff
@@ -612,78 +612,83 @@ To reduce the risk of synchronization across multiple senders,
+even if they do not exceed the path MTU as a result,
+can be detected by applications and could be ignored.
+This document does not define a mechanism to support detection,
+but one might be added in future.
+
 
 ## Monitoring Flows {#monitoring}
 
-Network elements can monitor flows
-to determine which flows are from applications
-that respect throughput advice.
-This section outlines an exemplary algorithm
-for network elements.
+Sending throughput advice is optional for any network.
+A network that sends throughput advice might, also optionally,
+choose to monitor flows
+to determine whether applications are following advice.
+
+This section outlines a method
+that a network element could use
+to determine whether advice is being followed.
+Network deployments that choose to monitor
+are free to follow any monitoring regime that suits their needs.
 
 This monitoring algorithm is guidance only.
-Network elements are advised that monitoring
-any more strictly than the following likely invalidates their advice.
-Network elements could choose not to monitor in this way
-or use a looser monitoring approach.
-For instance, monitoring over a longer time window
+However, monitoring any more strictly than the following
+could mean that an application
+might be incorrectly classified as not following advice.
+A looser monitoring approach,
+such as monitoring over a longer time window
 than the monitoring period (67s)
-or using a higher rate than is signaled
-is possible.
+or using a higher rate than is signaled,
+has no risk of incorrect classification.
 
-To operate this algorithm,
-the minimal state a network element maintains is:
-
-* the current throughput advice,
-* any state necessary to monitor throughput
-  (that is, throughput usage state, such as the state in {{sliding-window}}),
-* a timer used for rate increases, and
-* the time at which that throughput advice was last updated.
-
-When advice is set or updated,
-the network element waits until
-it receives the next SCONE packet on affected flows.
-It then updates the throughput advice in that packet ({{apply}}).
+When a network changes the value
+it intends to signal,
+applications need time to adjust their sending behavior.
+As a result, any monitoring needs to allow time
+for SCONE packets to be updated,
+for those packets to be received by endpoints,
+and for applications to adapt.
 
 A network element can then monitor affected flows
 to determine whether the throughput advice it provided
 was followed.
 
 A network element SHOULD base its monitoring
-on the maximum value it provided for throughput advice
-over at least the preceding monitoring period.
-Additional time might be allowed
-to account for the possibility that some SCONE packets might be lost.
+on the maximum value that was configured to apply
+during the preceding two monitoring periods.
+A longer period might be used
+to account for the possibility that SCONE packets are lost.
```

Comment:
IIRC, we assume 3 SCONE packets to be sent within one monitoring period. Therefore, it seems to me that this advice is not needed for network elements capable of updating every SCONE packet that they forward.

Assuming that is the case, I think we can drop this sentence, or change to the following if we are to close #75 along the way.

```suggestion
If the network element cannot update the throughput advice in every SCONE packet
(or can do so only infrequently), a longer period might be used
to account for the possibility that the updated SCONE packets are lost.
```